shuffle/cli/src/test.rs

```diff
@@ -107,7 +107,7 @@ pub fn run_deno_test_at_path(
         .join(",");
     let status = Command::new("deno")
         .args([
-            "test",
+            TEST_USERNAME,
```

Comment:
This isn't the `test` username, but rather, running the `deno test `cmd. Let's stick w the string literal here.